### PR TITLE
Add description of mount options to PersistentVolume describe printer

### PR DESCRIPTION
**Pull request by mlmhl**
**What this PR does / why we need it**:

Print `PersistentVolume.MountOptions` for `kubectl` describe command.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/printers/internalversion/describe.go [hunk 1]
```diff
@@ -1196,6 +1196,14 @@ func describePersistentVolume(pv *api.PersistentVolume, events *api.EventList) (
 		}
 		w.Write(LEVEL_0, "Reclaim Policy:\t%v\n", pv.Spec.PersistentVolumeReclaimPolicy)
 		w.Write(LEVEL_0, "Access Modes:\t%s\n", helper.GetAccessModesAsString(pv.Spec.AccessModes))
+		if len(pv.Spec.MountOptions) == 0 {
+			w.Write(LEVEL_0, "MountOptions:\t<none>\n")
+		} else {
+			w.Write(LEVEL_0, "MountOptions:\n")
+			for _, option := range pv.Spec.MountOptions {
+				w.Write(LEVEL_1, "%s\n", option)
```
Comment (janetkuo) [inline pkg/printers/internalversion/describe.go]: It should be fine to print all mount options in one line, i.e. `strings.Join(pv.Spec.MountOptions, ", ")`

**Review by janetkuo — COMMENTED** [review]
Would you add a test for it?

File: pkg/printers/internalversion/describe.go (hunk 1 above)
Comment (mlmhl) [reply pkg/printers/internalversion/describe.go]: Done, PTAL.

**Comment by mlmhl** [comment]
@janetkuo Tests added, PTAL.

**Review by janetkuo — APPROVED** [review]

**Comment by gnufied** [comment]
@mlmhl can you rebase please?

**Comment by mlmhl** [comment]
@gnufied  PR rebased :)

**Pull request closed without merging**